concourse/scripts/extract_quota_settings.rb

```diff
@@ -2,11 +2,15 @@
 require 'json'
 require 'yaml'
 
+output_dir=ARGV[0]
```

Comment:
Can we add some error handling to prevent it being run without a directory argument? At the moment it tries to write to the root directory:
```
➜  paas-cf git:(118837127_trade_tariff_quota) ✗ CF_MANIFEST=tests/cf-manifest.yml concourse/scripts/extract_quota_settings.rb
concourse/scripts/extract_quota_settings.rb:15:in `write': Permission denied @ rb_sysopen - /default_quota.sh (Errno::EACCES)
        from concourse/scripts/extract_quota_settings.rb:15:in `block in <main>'
        from concourse/scripts/extract_quota_settings.rb:10:in `each'
        from concourse/scripts/extract_quota_settings.rb:10:in `<main>'
```

`ARGV.fetch(0)` would work, but the feedback isn't very friendly. `raise "…" unless File.directory?` might be OK.